.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish a release
+
+on:
+  push:
+    tags:
+      - 'v**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get version
```

Comment:
That would not be necessary if the "human" action and the event were the release

Reply:
It is no longer worth since I need Get Version package to retrieve bupmed `package.json` in release action. It's more convenient than create it and store it along the workflow.